files/en-us/web/html/global_attributes/data-_star_/index.html

```diff
@@ -19,7 +19,7 @@
 
 <ul>
  <li>the name must not start with <code>xml</code>, whatever case is used for these letters;</li>
- <li>the name must not contain any semicolon (<code>U+003A</code>);</li>
+ <li>the name must not contain any semicolon (<code>U+003B</code>);</li>
```

Comment:
```suggestion
 <li>The name must not contain any colon characters (<code>:</code>).</li>
```
When submitting a patch like this one, you need to read the relevant spec requirements, and make sure the change aligns with the spec. In this case, the relevant spec requirements are here:

* https://html.spec.whatwg.org/multipage/dom.html#custom-data-attribute
* https://html.spec.whatwg.org/multipage/infrastructure.html#xml-compatible

Reply:
True, thanks for the patch. I assumed it has just the wrong Unicode.